pwiz_tools/Skyline/SettingsUI/PeptideGridViewDriver.cs

```diff
@@ -46,6 +46,7 @@ protected PeptideGridViewDriver(DataGridViewEx gridView, BindingSource bindingSo
         }
 
         protected bool AllowNegativeTime { get; set; }
+        protected bool AllowMissingTime { get; set; }

```

Comment:
Maybe a comment explaining why this is here?

Reply:
Added comment